packages/drivers/odsp-driver/src/compactSnapshotParser.ts

```diff
@@ -220,7 +214,6 @@ function readTreeSection(node: NodeCore): {
 			trees[path] = result.snapshotTree;
 			if (records.groupId !== undefined) {
 				const groupId = getStringInstance(records.groupId, "groupId should be a string");
-				// Non null asserting since trees[path] is already created
 				trees[path]!.groupId = groupId;
```

Comment:
non null assertion should be removed here as well?

Reply:
Thank you for the catch, the revert was corrupted, redid the revert